test/test_endpoints.py

```diff
@@ -396,22 +397,26 @@ def test_redirect_to_namespace(self):
 
 
 class OAuthTestCase(EndpointTestCase):
-    def test_authorize_nologin(self):
+    @parameterized.expand(["token", "code"])
```

Comment:
Could use pytest's existing parameterize?

Reply:
can't use `pytest.parameterize` because the test class is based on `unittest.Testcase` which doesn't support it